instrumentation/spring/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -419,8 +419,8 @@
     },
     {
       "name": "otel.instrumentation.logback-appender.experimental.capture-mdc-attributes",
-      "type": "java.util.List<java.lang.String>",
-      "description": "MDC attributes to capture. Use the wildcard character <code>*</code> to capture all attributes."
+      "type": "java.lang.String",
```

Comment:
Why it is no longer a list of String?

Reply:
It never technically was - it's not possible to use yaml lists